docs/reference/ilm/ilm-tutorial.asciidoc

```diff
@@ -89,13 +91,207 @@ For the complete list of actions that {ilm} can perform, see <<ilm-actions>>.
 
 [discrete]
 [[ilm-gs-apply-policy]]
-=== Create an index template to apply the lifecycle policy
+=== Create a composable template to create the data stream and apply the lifecycle policy
+
+To create a data stream we'll create a composable template with a `data_stream` definition
+that will specify the lifecycle policy.
+
+For example, you might create a `timeseries_template` that is will bootstrap a data stream
+named `timeseries`.
+
+To enable the ILM to manage the data stream, the template configures one {ilm-init} setting:
+
+* `index.lifecycle.name` specifies the name of the lifecycle policy to apply to the data stream.
+
+You can use the {kib} Create template wizard to add the template.
+This wizard invokes the put _index_template API to create the <<indices-templates,composable index template>>
+with the options you specify.
+
+The underlying request looks like this:
+
+[source,console]
+-----------------------
+PUT _index_template/timeseries_template
+{
+  "index_patterns": ["timeseries"],                 <1>
+  "data_stream": {
+    "timestamp_field": "@timestamp"                  <2>
+  },
+  "template": {
+    "settings": {
+      "number_of_shards": 1,
+      "number_of_replicas": 1,
+      "index.lifecycle.name": "timeseries_policy"     <3>
+    }
+  }
+}
+-----------------------
+// TEST[continued]
+
+<1> Apply the template when a document is indexed into the `timeseries` target.
+<2> Identifies the timestamp field for the data source, a field that must be present
+in all documents indexed into the `timeseries` data stream.
+<3> The name of the ILM policy used to manage the data stream.
+
+You can also invoke this API directly to add templates.
+
+[discrete]
+[[ilm-gs-bootstrap]]
+=== Bootstrap the data stream
```

Comment:
I'd say "create" rather than "bootstrap" here. Data streams kinda solve the original bootstrap problem so it'd be good to not use the same terminology.
```suggestion
[[ilm-gs-create-the-data-stream]]
=== Create the data stream
```